opppy/dump_utils.py

```diff
@@ -30,11 +30,18 @@
 import sys
 import pickle
 import math
-from multiprocessing import Process, Manager
+import platform
+from multiprocessing import Process, Manager, cpu_count
 
 from opppy.progress import progress
 
 USE_THREADS = os.getenv("OPPPY_USE_THREADS", 'True').lower() in ('true', '1', 't')
+NTHREADS = int(os.getenv("OPPPY_N_THREADS", str(min(cpu_count(),4))))
+# Protect against multiprocessing fork issue on Windows
+if "windows" in platform.system().lower(): 
```

Comment:
@salvesen-lanl I'm hoping this is windows only issue and that this fix up avoids the issue for now. If you are in fact on a windows system, can you run your tests again with this change set included.